iib/web/api_v1.py

```diff
@@ -396,6 +396,40 @@ def get_builds() -> flask.Response:
         query_params['user'] = user
         query = query.join(Request.user).filter(User.username == user)
 
+    if from_index:
```

Comment:
I think a lot of code is similar to that of index_image. Can I please ask if it would be possible to create a helper function instead? Or de-duplicate some of the code like the aliases?